src/components/Auth/UserFormDialog/controls/EmailPassword.test.tsx

```diff
@@ -74,9 +57,33 @@ describe('EmailPassword', () => {
       await triggerValidation();
       expect(queryByText(errorText)).not.toBeNull();
     });
+
+    it('invalid for duplicate email', async () => {
+      const { getByText, triggerValidation } = setup(
+        {
+          email: validEmail,
+          password: 'lollol',
+        },
+        { allEmails: new Set([validEmail]) }
```

Comment:
I'm not sure client-side validation is the right approach. As we move on with pagination and only getting part of the results from the backend, this will not catch all issues and we'd need to handle duplicate email server error anyway. With that being said, this is probably fine for v1.

Reply:
Once we have pagination, we should be able to make this async and just send requests to the server?

Reply:
Yeah, good point. I don't know if react-form can handle async validation, but we can figure this out later. API side, we just need to query users by the exact email. There may be race conditions, but I guess as long as we have a proper way to catch generic errors, we should be all good.